bwa.wdl

```diff
@@ -44,6 +44,53 @@ task Mem {
     }
 }
 
+task Kit {
+    input {
+        File read1
+        File? read2
+        BwaIndex bwaIndex
+        String outputPrefix
+        String? readgroup
+        Boolean sixtyFour = false
+
+        Int threads = 2
+        Int sortThreads = 2
+        Int memory = 10
+        String dockerImage = "biocontainers/bwakit:v0.7.15_cv1"
+    }
+
+    command {
+        set -e
+        bwa mem \
+          -t ~{threads} \
+          ~{"-R '" + readgroup}~{true="'" false="" defined(readgroup)} \
+          ~{bwaIndex.fastaFile} \
+          ~{read1} \
+          ~{read2} \
+          2> ~{outputPrefix}.log.bwamem | \
+        k8 /opt/conda/bin/bwa-postalt.js \
+          -p ~{outputPrefix}.hla \
+          ~{bwaIndex.fastaFile}~{true=".64.alt" false=".alt" sixtyFour} | \
+        samtools sort \
+          -@ ~{sortThreads} \
+          -m1G \
+          - \
```

Comment:
I am not very familiar with samtools, but a empty dash seemed strange to me, does this have a specific purpose?

Reply:
It means "read from stdin". http://www.htslib.org/doc/samtools.html